lib/util.js

```diff
@@ -1366,4 +1377,3 @@ module.exports = {
     return item;
   }
 };
-
```

Comment:
thanks for the changes! I know what I am gonna say is stupid but there is not need to delete this line 👍 doing amazing job!

Reply:
Completely fair, wasn't an intentional change so I suspect some editor wierdness (usually all set up to leave a single line at the end of a file)

Reply:
thanks for the great job and great communication! keep it up

Reply:
TLDR: I don't know why the line went...but it should be there...please double check you don't end up with 2 when this is merged.



So what's really odd is that in my local codebase the eof line is there, and `npm run test-lint` passes (minus 5 unrelated warnings).

But on the fork it appeared to be missing 😕 , except when editing the file online when it appears 😖 , but I had to add a second eof line for it to get rid of the change..... 😱

Reply:
GitHub voodoo